src/components/text-area/daikin-text-area.ts

```diff
@@ -0,0 +1,204 @@
+    "disabled:text-[#BFBFBF]",
+    "disabled:bg-white",
+    "disabled:placeholder:text-[#BFBFBF]",
+  ],
+  {
+    variants: {
+      error: {
+        false: [
+          "enabled:var-color-[#515151]/color-base",
+          "focus-visible:var-color-[#0081C0]/color-state-focus",
+        ],
+        true: ["enabled:var-color-[#D80C18]/color-base"],
+      },
+      resize: {
+        false: ["resize-none"],
+        true: ["resize-vertical"],
+      },
+    },
+  }
+);
+
+/**
+ * The text area component is designed for multiline text input, similar to the HTML `<textarea>` tag.
+ * It is ideal for situations where users need to enter longer pieces of text, such as comments, descriptions, or messages.
+ * Can be used within `daikin-input-group` component.
+ *
+ * Hierarchies:
+ * - `daikin-text-area` (can be used solely)
+ * - `daikin-input-group` > `daikin-text-area`
+ *
+ * @fires change - A cloned event of a [change event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) emitted from the inner `<textarea>` element.
+ * @fires input - A retargeted event of a [input event](https://developer.mozilla.org/en-US/docs/Web/API/Element/input_event).
+ *
+ * @example
+ *
+ * ```js
+ * import "@daikin-oss/design-system-web-components/components/text-area/index.js";
+ * ```
+ *
+ * ```html
+ * <!-- See `daikin-input-group` component for complete example. -->
+ * <daikin-text-area name="name"></daikin-text-area>
+ * ```
+ */
+@customElement("daikin-text-area")
+export class DaikinTextArea extends LitElement {
+  static override readonly styles = css`
+    ${unsafeCSS(tailwindStyles)}
+
+    :host {
+      display: block;
+      height: 87px;
+      position: relative;
+    }
+  `;
+
+  static readonly formAssociated = true;
+
+  private _internals = this.attachInternals();
+
+  /**
+   * Value of the text area.
+   */
+  @property({ type: String })
+  value = "";
+
+  /**
+   * Form name of the text area.
+   */
+  @property({ type: String, reflect: true })
+  name = "";
+
+  /**
+   * Placeholder text.
+   */
+  @property({ type: String, reflect: true })
+  placeholder: string | null = null;
+
+  /**
+   * Whether the text area is disabled.
+   */
+  @property({ type: Boolean, reflect: true })
+  disabled = false;
+
+  /**
+   * Whether the text area is readonly.
+   */
+  @property({ type: Boolean, reflect: true })
+  readonly = false;
+
+  /**
+   * Whether the text area is required.
+   */
+  @property({ type: Boolean, reflect: true })
+  required = false;
+
+  /**
+   * Error state. Ignored if the `disabled` is `true`.
+   */
+  @property({ type: Boolean, reflect: true })
+  error = false;
+
+  /**
+   * Value of `autocomplete` attribute of the internal `<textarea>`.
+   */
+  @property({ type: String, reflect: true })
+  autocomplete?: HTMLInputElement["autocomplete"];
+
+  /**
+   * Whether to allow resizing of the text area.
+   */
+  @property({ type: Boolean, reflect: true })
+  resizable = false;
```

Comment:
Let's move the counting into a getter so that we can move to a more accurate logic than `value.length` in the future.

```suggestion
  resizable = false;

  get count(): number {
    return this.value.length;
  }
```